src/messaging/ExchangeContext.cpp

```diff
@@ -54,6 +54,8 @@ using namespace chip::Encoding;
 using namespace chip::Inet;
 using namespace chip::System;
 
+#define CHIP_EXCHANGE_CONTEXT_DETAIL_LOGGING 1
```

Comment:
Is this meant to be checked in?

Reply:
I just placed it here for now so that I can debug any test failures that happen in CI. Once they all pass, I'll be removing this.